backend/server.js

```diff
@@ -18,7 +70,70 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  const endpoints = expressListEndpoints(app);
+  res.json(endpoints);
+});
+
+app.post("/register", async (req, res) => {
```

Comment:
Code looks good! Just remember to try to name endpoints after what they return or in this case create. For example: /users